README.md

```diff
@@ -67,6 +67,16 @@ Therefore it is recommended to clone the training repository in the Linux filesy
 > installing packages from GitHub, resulting in failures to update sensei
 > correctly.
 
+⚠️ On Mac M1 or M2, you need to follow special steps to enable PDF generation:
```

Comment:
Shouldn't it be done *before* installing sensei? Otherwise, as puppeteer is a dependency of sensei, it will be installed with embedded chromium.